GetRefreshToken for token refresh

johannaratliff opened the pull request:
using client_id and existing refresh token

Signed-off-by: Mark Alston <malston@pivotal.io>

Commit:
GetRefreshToken for token refresh against /oauth/token

Signed-off-by: Mark Alston <malston@pivotal.io>

Commit:
Clean up commented out BeforeEach loops

Signed-off-by: Johanna Smith <josmith@pivotal.io>

poy commented:
`GetRefreshToken()` only returns a refresh token, however the UAA returns a refresh token and access token.

Commit:
Add access token to response for refresh token

Commit:
Add sample for getting a new refresh token

Commit:
Add token type to access token

- Access token from refresh token should include bearer
- refresh tokens should not be prefixed with bearer

- delete http tests as they provided no additional coverage

Signed-off-by: Travis Patterson <tpatterson@pivotal.io>

MasslessParticle commented:
Manually merged
